### PR TITLE
Remove numpy from PEP 518 build deps

### PR DESCRIPTION
**Pull request by auscompgeek**
We assume OpenCV has been installed with the Python 3 bindings, so this
should be a safe assumption I guess.

We should test the typical user's install flow.

### CODE REVIEW

**Comment by auscompgeek** [comment]
Opened pybind/pybind11#1644 for the resulting build failure.

File: setup.py [hunk 1]
```diff
@@ -259,7 +241,7 @@ def get_libuv_sources(d):
     long_description=long_description,
     packages=find_packages(),
     ext_modules=ext_modules,
-    setup_requires=["numpy", "pybind11>=2.2"],
+    setup_requires=["numpy"],
```
Comment (virtuald) [inline setup.py]: Is this needed then?

Comment (auscompgeek) [reply setup.py]: :man_shrugging:

Comment (virtuald) [reply setup.py]: Let's remove it.

**Pull request merged**